internal/xds/resolver/xds_resolver_test.go

```diff
@@ -411,9 +411,6 @@ func (s) TestResolverBadServiceUpdate_NACKedWithCache(t *testing.T) {
 	// Expect an error update from the resolver. Since the resource is cached,
```

Comment:
This comment probably needs to be updated since the resolver doesn't report an error for ambient listener and route errors anymore.